stack.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+
+class Element(object):
+    ''' Create data element with default value and previous pointer. '''
+    def __init__(self, value=None, previous=None):
+        ''' Value and previous pointer default to none. '''
+        self.val = value
+        self.previous = previous
+
+
+class Stack(object):
+    ''' Create an empty stack. '''
+    def __init__(self):
+        self.top = None
+
+    def push(self, element):
+        ''' Add data element to the top of stack. '''
+        self.top, self.top.previous = element, self.top
```

Comment:
As I stated in class, we changed this to require that you put a value in and get a value out.  Could you please update it?